### DISCUSSION TITLE
Integration with Audacity

### DISCUSSION THREAD

**Discussion by fritterandwaste** [hydrogen-music/hydrogen#1994] [Ideas]
This is perhaps a bit of a pipedream but it would be so helpful if Hydrogen could be included as an 'add-on' to Audacity and integrated in a way the editing could be done within an Audacity track. Currently I have to make educated guesses as to timings, export to .wav, import to Audacity and play to establish if I've got it right and repeat the process if not. 

I appreciate that the existing ability to embed a playback track achieves something similar if not quite as advanced but might it be possible add a feature to drag the desired playback start position or allow an offset to be set in seconds (with 2 decimal places) as to where it should start?

Many thanks for what's there already - I use it a lot!

**Comment by theGreatWhiteShark** [comment]
Hey @fritterandwaste

You can use [JACK](https://jackaudio.org/) to sync Hydrogen and Audacity. In addition to align transport/playback, you can also use it to pipe Hydrogen's audio output directly into Audacity without the need to export it first (see https://manual.audacityteam.org/man/tutorial_recording_computer_playback_on_linux.html).

**Comment by fritterandwaste** [comment]
Many thanks for that. Sounds brilliant but it could be a bit of a challenge for me as a mere mortal.

I note the reference to Linux but is it also possible in Windows?

Thanks again.

**Reply by theGreatWhiteShark** [reply]
>  note the reference to Linux but is it also possible in Windows?

I hope so. At least it worked fine for quite a while. But there have been a couple complaints about recent versions and I'm afraid you have to check for yourself whether it is working at your end. (I'm not a Windows user and have not tested it.)

Alternatively, there are Linux distros dedicated to multimedia, like [Ubuntu Studio](https://ubuntustudio.org/) and [AV Linux](https://www.bandshed.net/avlinux/). Those have the benefit that all audio stuff is more or less plug and play and works out of the box without requiring you to know (merely) anything about Linux. You could try those using a Live USB stick.
